slices/libpam-runtime.yaml

```
@@ -0,0 +1,60 @@
+package: libpam-runtime
+
+slices:
+  bins:
+    essential:
+      - libpam-modules_libs
+    contents:
+      # perl scripts that rely on debconf to properly work and discover
+      # plugins/rules in /usr/share/pam-configs/*
+      /usr/sbin/pam-auth-update:
+      /usr/sbin/pam_getenv:
+
+  config:
+    contents:
+      /etc/pam.conf:
+      /etc/pam.d/other:
+  
+  # folders expected by libpam to exist
+  folders:
```

I am not sure if this is a better name or not. But `folders` sound a bit weird. Let me know if you have other suggestions!
```suggestion
  var:
```